src/NuGet.Clients/NuGet.VisualStudio.Common/VsHierarchyUtility.cs

```diff
@@ -77,7 +77,9 @@ public static bool IsProjectCapabilityCompliant(IVsHierarchy hierarchy)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            return hierarchy.IsCapabilityMatch("AssemblyReferences + DeclaredSourceItems + UserSourceItems");
+            // NOTE: (AssemblyReferences + DeclaredSourceItems + UserSourceItems) exists solely for compatibility reasons
+            // with existing custom CPS-based projects that existed before "PackageReferences" capability was introduced.
+            return hierarchy.IsCapabilityMatch("(AssemblyReferences + DeclaredSourceItems + UserSourceItems) | PackageReferences");
```

Comment:
We have another capability check, [see here](https://github.com/NuGet/NuGet.Client/blob/15777600efa6f37fb336c9844177d361a8568e00/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProjectProvider.cs#L78). 

That one probably needs to be a PackageReferences check but I think it can break some custom project-systems if any.

Reply:
I think this is good enough for now though. 
Happy to have a discussion if necessary @davkean